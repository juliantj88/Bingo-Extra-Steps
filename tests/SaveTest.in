1
test1
test2
1 R 3
2 B 2
5 R 2
4 U 3
3 Y 4
0 Y 4
0 B 1
0 U 5
0 L 5
save SaveTest
4