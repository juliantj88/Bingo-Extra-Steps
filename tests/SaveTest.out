test1
test2
3
1
0
F...............
RBUU
RRLL
RBBU
BYYU
RRYU
Byrul
lbyRu
ulbyR
rulby
yrulb
byrul
lByru
Ulbyr
rulby
yrulb
.
..
...
YYY.
LL...
.
..
...
YY..
UU...
       
       

RRLBBYRLLUYBURRBLRYRBLBUUBRLBLULYRLYLLUYYULLLRLUYBLBYYYRBUURUBBYUB
0